data/srfi-data.scm

```diff
@@ -2171,12 +2171,13 @@
  (keywords data-structure)
  (draft-date "2023-11-14"))
 ((number 251)
- (status draft)
+ (status final)
  (title "Mixing groups of definitions with expressions within bodies")
  (author "Sergei Egorov")
  (see-also 245)
  (keywords binding)
- (draft-date "2023-11-30"))
+ (draft-date "2023-11-30")
```

Comment:
Same as above.

Reply:
That file got updated while regenerating the table. I think it takes the values from the upstream srfi-data.scm.

I updated the title of this PR to indicate that the data is being updated as well.

Reply:
I just pushed a commit to update srfi-data.scm.

Reply:
Oh, I see.  Sorry about that.  I mistook this for the copy in srfi-common.  But since it's a copy, shouldn't it be in .gitignore?

Reply:
It probably should be.

Reply:
I removed the intermediate files and added a .gitignore.